build/Dockerfile

```diff
@@ -18,6 +18,7 @@ ENV NVIDIA_DRIVER_CAPABILITIES=utility
 ENV NVIDIA_MIG_MONITOR_DEVICES=all
 ENV NVIDIA_MIG_CONFIG_DEVICES=all
 
+RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical && yum clean all
```

Comment:
with `:latest` image, do we need the yum update?

Reply:
A good question, and for safety, I would like to say yes.
ref: https://catalog.redhat.com/software/containers/ubi9/618326f8c0d15aff4912fe0b?architecture=amd64&image=65e093e60a21b531a96f93ca&container-tabs=security
taking today's latest image as an example, even if the tag is latest, the new CVE comes after the image been created. So there is a time window, which yum can fix some CVE for us.
On the other hand, as the CVE is related with any package, impart us or not.

Therefore.... we hope the yum install can help as table below during the time window according to severity level.

|             | Related to us | no related |
|-------------|---------------|------------|
| Critical    | Fixed         | Fixed      |
| Important   | Fixed         | Fixed      |
| High or low | n/A           | n/A        |